devtool/src/main.rs

```diff
@@ -167,6 +167,19 @@ fn task_package() {
         format!("{}/bin/jjs-db-init", &pkg_dir),
     )
     .unwrap();
+    let opts = fs_extra::dir::CopyOptions {
```

Comment:
why not fs_extra::dir::CopyOptions::new()?

Reply:
Copy-pasted from later code.